src/asciidoc/reference/packages/withkojiVCC.adoc

```diff
@@ -282,6 +301,8 @@ import { FeedSdk } from '@withkoji/vcc';
 const feed = new FeedSdk();
 ----
 
```

Comment:
We need to add info about contacting Koji to enable the feed functionality, e.g., the last line of the current readme (needs some editing but this is the gist of it)...
"When you are confident that your app is able to function well within a feed, please contact us and we will review and authorize it."